.github/workflows/main.yaml

```diff
@@ -56,21 +56,34 @@ jobs:
     strategy:
       matrix:
         build:
-          - jammy
-          - noble
+          - jammy-amd64
+          - noble-amd64
```

Comment:
why are here the aarch64 variants missing? If it is not required by GHA workflows, then remove the while `build` list.